molgenis-dataexplorer/src/main/resources/js/dataexplorer-data.js

```diff
@@ -92,7 +92,8 @@
 			query : {
 				rules : [entityQuery.q]
 			},
-			colNames : $('input[name=colNames]:checked').val()
+			colNames : $('input[name=colNames]:checked').val(),
+			downloadType : $('input[name=downloadTypes]:checked').val()
 		};
```

Comment:
Galaxy export is a different button, but yes it is always csv. For excel the same thing happens as with csv, if someone chooses labels or names the column headers change